modules/nf-core/metamaps/mapdirectly/main.nf

```diff
@@ -0,0 +1,42 @@
+process METAMAPS_MAPDIRECTLY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metamaps:0.1.633d2e0--h21ec9f0_0':
+        'biocontainers/metamaps:0.1.633d2e0--h21ec9f0_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path database
+
+    output:
+    tuple val(meta), path("*classification_res")         , emit: classification_res
+    path("*classification_res.meta")                     , emit: meta_file
+    path("*classification_res.meta.unmappedReadsLengths"), emit: meta_unmappedreadsLengths
+    path("*classification_res.parameters")               , emit: para_file
```

Comment:
Should these also have a meta associate with them?

Reply:
I have added the meta now.